### PR TITLE
completing post path for comments and tests

### PR DESCRIPTION
**Pull request by libertyskies**
completing post path for a new comment and testing

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -236,7 +236,89 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+    const testComment = {
+      username: "noSuchUsername",
+      body: "The wonderful thing about Tiggers is that I'm the only one",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(testComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("username not found");
```
Comment (roisinoreilly) [inline __tests__/app.test.js:321]: excellent tests, great job

File: models/comments.model.js [hunk 2]
```diff
@@ -8,10 +8,31 @@ exports.fetchCommentsByArticleId = async (id) => {
...
-  await checkIdExists("articles", "article_id", id);
+  await checkValueExists("articles", "article_id", id);
 
   const values = [id];
   const query = `SELECT comment_id, votes, created_at, author, body, article_id FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`;
   const { rows } = await db.query(query, values);
   return rows;
 };
+
+exports.createComment = async (comment, id) => {
```
Comment (roisinoreilly) [inline models/comments.model.js:19]: really nicely done here

**Review by roisinoreilly — APPROVED** [review]
excellent job with this ticket, your code is really neat and readable and your testing is very good. PR approved! keep it up :)

File: __tests__/app.test.js [hunk 3]
```diff
@@ -236,7 +236,89 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+    const newComment = {
+      username: "rogersop",
+      body: "The wonderful thing about Tiggers is that I'm the only one",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201);
+  });
+  test("returns a 201 status code when passed a new comment with the correct properties", () => {
```
[the remark this answers is not in the document]
Comment (libertyskies) [reply __tests__/app.test.js]: thank you, edited to update test description

**Pull request merged**